.storybook/main.js

```diff
@@ -108,3 +110,25 @@ function overrideDefaultBabelLoader(rules) {
 
   loader.options.customize = customLoaderPath;
 }
+
+/**
+ * @returns {import('storybook-addon-export-to-codesandbox').BabelPluginOptions}
+ */
+function getCodesandboxBabelOptions() {
+  const allPackageInfo = getAllPackageInfo();
+
+  /** @type {import('storybook-addon-export-to-codesandbox').BabelPluginOptions}  */
+  const initialValue = {};
+  return Object.values(allPackageInfo).reduce((acc, cur) => {
+    if (isConvergedPackage(cur.packageJson)) {
+      const prereleaseTags = semver.prerelease(cur.packageJson.version);
+      if (prereleaseTags && !prereleaseTags[0].includes('rc')) {
```

Comment:
when we will ship stable this will need to be manually changed ?

Reply:
not at all the condition is rather if prerelease tags exist and they are not `rc` which should only cover `beta`  and `alpha`

Reply:
I've modified the code to make this point clearer